build.sbt

```diff
@@ -226,8 +225,7 @@ lazy val pluginTesterJava = Project(id = "akka-grpc-plugin-tester-java", base =
     ReflectiveCodeGen.generatedLanguages := Seq("Java"),
     crossScalaVersions := Dependencies.Versions.CrossScalaForLib,
     scalaVersion := scala212,
-    ReflectiveCodeGen.codeGeneratorSettings ++= Seq("server_power_apis"),
-    Compile / ReflectiveCodeGen.protocOptions += "--include_std_types")
```

Comment:
why did you remove this here? did it become the default behavior?

Reply:
I tried to find something in the docs, but didn't succeed.
`protoc` doesn't accept the flag anymore.